scenario/3_struct_mapping_extend.yml

```diff
@@ -0,0 +1,45 @@
+input:
+    input.go: |
+        package structs
+
+        // goverter:converter
+        // goverter:mapExtend PluckAgeInt
+        // goverter:mapExtend PlusScoreInt
```

Comment:
This shouldn't be needed, defining it inside on the converter method should be enough.

Reply:
remove mapExtend parse in converter comment https://github.com/jmattheis/goverter/pull/25/commits/5b8e6df972144c269ed780cd1e060af8fe385d2b